File: SecondYear/ApoorvaSri123_ApoorvaSrivastava_2226it1116_2/Week_2/pow.c
#include <stdio.h>
#include <math.h>

double calculatePower(double x, int n);

int main() {
    double x;
    int n;
    printf("Enter the base (x): ");
    scanf("%lf", &x);
    printf("Enter the exponent (n): ");
    scanf("%d", &n);
    
    double result = calculatePower(x, n);
    printf("%lf\n", result);
    return 0;
}

double calculatePower(double x, int n) {
    return pow(x, n);
}

File: SecondYear/praveer7398_PraveerNandan_2226ce1011_2/Week_2/pow.java
import java.util.*;
public class pow {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            double x = sc.nextDouble(); 
            int n = sc.nextInt();     
            double result = calculatePower(x, n);
            System.out.println(result+"00000");
        }
    }

    public static double calculatePower(double x, int n) {
        return Math.pow(x, n);
    }
}
